swan/utils/early_stopping.py

```diff
@@ -25,6 +27,19 @@ def __init__(self, patience: int = 7, delta: float = 1e-6):
         self.delta = delta
 
     def __call__(self, saver: Callable, epoch: int, val_loss: float):
```

Comment:
```suggestion
    def __call__(self, saver: Callable, epoch: int, val_loss: float) -> None:
```
FYI a function without an explicit return-type is equivalent to returning `Any`.